pkg/cloudprovider/providers/gce/gce_disks.go

```diff
@@ -809,6 +862,48 @@ func (gce *GCECloud) DeleteDisk(diskToDelete string) error {
 	return err
 }
 
+func (gce *GCECloud) ResizeDisk(diskToResize string, oldSize resource.Quantity, newSize resource.Quantity) (resource.Quantity, error) {
+	disk, err := gce.GetDiskByNameUnknownZone(diskToResize)
```

Comment:
Do we not have the disk zone information by now?

Reply:
Not necessarily, since we started to support regional disks, in which case zone information may be unknown. The regional disks are alpha feature though.

Reply:
For zonal PDs, you could use the same name across zones, so this call would fail in that case.  But when we provision PDs, the names should be unique, so maybe not an issue.

Reply:
I used the same flow we use for Deleting the disks btw. I think, we should be fine.